microsetta_admin/templates/scan_bulk.html

```diff
@@ -0,0 +1,203 @@
+{% extends "sitebase.html" %}
+{% block head %}
+<link rel="stylesheet" type="text/css" href="/static/css/table_style.css">
+<script type="text/javascript" language="javascript" src="/static/js/ruleset.js"></script>
+<!--<link rel="stylesheet" type="text/css" href="/static/vendor/DataTables/DataTables-1.10.21/css/jquery.dataTables.css">-->
+<script src="/static/vendor/DataTables/DataTables-1.10.21/js/jquery.dataTables.js"></script>
+<script src="/static/vendor/DataTables/Buttons-1.6.2/js/dataTables.buttons.min.js"></script>
+<script src="/static/vendor/DataTables/Buttons-1.6.2/js/buttons.html5.min.js"></script>
+<script>
+    function remove_error_messages () {
+        block = document.getElementById("search_errors");
+        if (block !== null) {
+            block.innerHTML = "";
+        }
+    };
+        $(document).ready(function() {
+        {% if resource is not none %}
+            $('#search_results').DataTable({
+                // more information on "dom" can be found here:
+                // https://datatables.net/reference/option/dom
+                dom: 'Bfrtip',
+                buttons: ['csv']
+                }
+            );
+        {% endif %}
+    } );
+    // prevent submit with dummy status
+    function verify_status() {
+        let result = true;
+        let select_element = document.getElementById("sample_status");
+        let selected_status = select_element.options[select_element.selectedIndex].text;
+        if (selected_status === '{{ dummy_status }}'){
+            result = false;
+            alert("A scan cannot be saved without selecting a sample_status.");
+        }
+
+        return result;
+    }
+
+    $(function(){
+        document.search_form.sample_barcode.focus();
+
+        old_status = new NamedExpression("OLD_SAMPLE_STATUS", "{{latest_status}}")
```

Comment:
I didn't see it, but perhaps missed it, where in biocore/microsetta-private-api#496 sample status or other details about a barcode are returned?